debian/control

```diff
@@ -0,0 +1,431 @@
+           mariadb-server-5.3,
+           mariadb-server-5.5,
+           mariadb-tokudb-engine-10.0,
+           mariadb-tokudb-engine-5.5,
+           mysql-server (<< ${source:Version}),
+           mysql-server-4.1,
+           mysql-server-5.0,
+           mysql-server-5.1,
+           mysql-server-5.5,
+           mysql-server-5.6,
+           virtual-mysql-server
+Replaces: libmariadbclient-dev (<< 5.5.0),
+          libmariadbclient16 (<< 5.3.4),
+          mariadb-server (<< ${source:Version}),
+          mariadb-server-10.0,
+          mariadb-server-5.1,
+          mariadb-server-5.2,
+          mariadb-server-5.3,
+          mariadb-server-5.5,
+          mariadb-tokudb-engine-10.0,
+          mariadb-tokudb-engine-5.5,
+          mysql-server (<< ${source:Version}),
+          mysql-server-4.1,
+          mysql-server-5.0,
+          mysql-server-5.1,
+          mysql-server-5.5,
+          mysql-server-5.6,
+          virtual-mysql-server
+Description: MariaDB database server binaries
+ MariaDB is a fast, stable and true multi-user, multi-threaded SQL database
+ server. SQL (Structured Query Language) is the most popular database query
+ language in the world. The main goals of MariaDB are speed, robustness and
+ ease of use.
+ .
+ This package includes the server binaries.
+
+Package: mariadb-server
+Architecture: all
+Depends: mariadb-server-10.1 (= ${source:Version}), ${misc:Depends}
+Description: MariaDB database server (metapackage depending on the latest version)
+ This is an empty package that depends on the current "best" version of
+ mariadb-server (currently mariadb-server-10.1), as determined by the MariaDB
+ maintainers. Install this package if in doubt about which MariaDB
+ version you need. That will install the version recommended by the
+ package maintainers.
+ .
+ MariaDB is a fast, stable and true multi-user, multi-threaded SQL database
+ server. SQL (Structured Query Language) is the most popular database query
+ language in the world. The main goals of MariaDB are speed, robustness and
+ ease of use.
+
+Package: mariadb-client
+Architecture: all
+Depends: mariadb-client-10.1 (= ${source:Version}), ${misc:Depends}
+Description: MariaDB database client (metapackage depending on the latest version)
+ This is an empty package that depends on the current "best" version of
+ mariadb-client (currently mariadb-client-10.1), as determined by the MariaDB
+ maintainers.  Install this package if in doubt about which MariaDB version
+ you want, as this is the one considered to be in the best shape.
+
+Package: mariadb-test
+Architecture: all
+Depends: mariadb-test-10.1 (= ${source:Version}), ${misc:Depends}
+Description: MariaDB database regression test suite (metapackage for the latest version)
+ This is an empty package that depends on the current "best" version of
+ mariadb-test (currently mariadb-test-10.1), as determined by the MariaDB
+ maintainers.
+
+Package: mariadb-connect-engine-10.1
+Architecture: any
```

Comment:
should be renamed to mariadb-connect-engine
because the plugin version should always match the server
version and the user should not be able to pin
plugin to 10.0

same for all plugins and for mariadb-tests-10.0